### ISSUE TITLE
Bypass 10 App ID limit for free ADP members by using "Universal" App IDs

### ISSUE THREAD

**Issue by JJTech0130** [SideStore/SideStore#68]
<div dir="auto" class="current markeddown hide-on-edit js-desc js-show-with-desc"><p dir="auto"><strong>Related</strong><br>While working on this, consider switching to Apple ID suffixes rather than Team ID for reasons explained here: <a href="https://trello.com/c/yWu2VRiW/95-change-altstores-appended-suffix-to-the-apple-id-instead-of-the-team-id-when-it-resigns-ipas" class="known-service-link"><img class="known-service-icon" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAEz0lEQVR4Xu1bf0xVVRz/3Ccg5UtwQkDI9sIpvTlQrAUzyx9tsnQtsQyZNqVhuKLAZ1uSmNBAzQV7VmoIDTSb+COhqWywLFzqpBDXqLEs4TVEJHDx6mnweLzXuQ8xfl3Ofdz77u7rnfPne+d8P9/zOef7vd9zvt/DQajNji5CVHQiImYGQzerB9MftuOhAD/4+PoIjlHDH7Y+G/42W3H7Dw1Mv/qj9XonfmmswLXGtLHU40b9OH/BNixMyMPiFWqYjnw61J4FLlRno+FS/lChwwlIeKkZKYYZ8PPzlQ9ZRZKs1j6UFt5A9cnIQa3+I2D5mhakvq1TkbruU6XkQxOqyh/lAQYIiF+6A4b8bNXbt1yU8H7CuD0Pl77OHSDAsNOBhcvkEu8Zci7UAIXvchz0scXIK0oBp5nkGZrLpKXD3o/stFIOyZu6sDp1ukxiPUvMiZLbHLYWOPDkIvGKt7UAHTeB9huAxSw8ThsAhM0AQsKB8An6VndjETPgsPd4LyIi/cQwEHz0E7yRsARx+tmI1esRMo1MUqB1/GnG1aYm1DVdw77qb9GZnC4G4n4fXUUxdiQlQhcSjHmPRSFQO2Xc8bX1Dei+cxebPzsCU+JGcVitzVYOh8/1QksiPEpLuXIGH23PgnYytesoSZZeKwy79qB47nM0GOf/G+pPozQ/R1TfkZ267/Zgc/5ulD3xPH28xUwIOHHZhkk+4zpAfuV/O/wppvpPpgsV6NFrsyEqJR2/rxp/dfiVbyFYUhpPQuymTPpO6Lf1czhV7xgXrM0EYzCQkfSiFJ2cY/d9+RXS2/uIT3DGIKMbwaqMCcULzzwlGetQVQ02NP8ljHUPgU5Aw0VcfvlZxEXPkaxUXePPiD9+DpgvMEGCVbvyaSx6PFYy1vkrV7G48jthLNEEnD2G7r05CJjyoGSlzMRJBWYQ216RNLYsguUo2SMZhxfQbbmDaZm5wliiCTh2EI7yIlmU4oVwa8ipNOm1seUpicUIGGCA7gOUXBUlsdgOYDuAmQDzAcwJsq8A+wyyOIB6GlQyOFESiwVCLBBigRALhFggpLZASMYbIertkyq/Aqq8E1Ty20wyQZUxYTLdCleTW2ELNSulrkiQbEs58gLmf3owL01EXkB1TvCeXUrJDPGTz8wTmRlSKwE8D/xOMKa+gkByHc/nBmnX8nwewNTRiZzyU/SM0JA7adWZwLD7cpIpQkebC5noR6iZoJH38eomQLZshLAgRoCqjsMKrDgzgREMMBNgJkArkFAyFGY+QHkG6D5AyQIJ5ecvIi+gZImMKgkgR9SyyKlYv1x6LfGhKnFHVCV5oJsA0Sa+9iQuGndCQ0qLJ9rsDgcWZGShbsnqiYpwyzhRBPDIbzZ9g8KtW+Cj0bisiM1uh2F3AT7WL3V5rLsHiCqUHFRibs0X+GB9MmaGhyE0KAjaB/wF9bOQc/mtri5cv9mOd8qO4sdla909F9flOwslRZbK3peuwBHV9ZlMcISzVHZ/hRWhEf/PN0I0Xm619rleLk8T6kn/f3+exAGrXu3EuteDPElv2XQ9sr+Lgy7qAAo+3+iVT2a2rD3IHk05t1NMXBa2GXPh6+sdzpB/Nvf+W+/hpx92sYeTwxyKVz+dHWTCqx9PD90OXvJ8/l9pKQluh4tO7AAAAABJRU5ErkJggg==">ENHANCEMENT: Change AltStore’s app ID suffix to "AppleID.TeamID" instead of just TeamID to allow for seamless paid dev team switching *on device*</a></p>
<p dir="auto"><strong>Premise</strong><br>Using "Universal App IDs" that will essentially eliminate the issue caused by the 10 App ID limit.</p>
<p dir="auto">Limitations to this per IPA could be specified in the JSON source (as those are developer-maintained) given not all IPAs can have their ID changed and still function properly. For those, AltStore can sign them using a universal App ID and install them with whatever the Bundle ID needs to be. </p>
<p dir="auto"><strong>Reference</strong><br><a href="https://discordapp.com/channels/625766896230334465/630158257910185994/710038218279616514" target="_blank" rel="noreferrer ugc nofollow" class="atlaskit-smart-link"><div class="js-react-root"><span class="loader-wrapper"><a href="https://discordapp.com/channels/625766896230334465/630158257910185994/710038218279616514" tabindex="0" role="button" data-testid="inline-card-resolved-view" class="css-180f5r3"><span class="css-1awfwlv"><span class="css-1n3444m"><span class="css-w2w4jx"></span><span class="css-14y9yja"><span data-testid="inline-card-icon-and-title-default" role="img" aria-label="link" style="--icon-primary-color: currentColor; --icon-secondary-color: var(--ds-surface, #FFFFFF);" class="css-hakgx8"><svg width="24" height="24" viewBox="0 0 24 24" role="presentation"><g fill="currentColor" fill-rule="evenodd"><path d="M12.856 5.457l-.937.92a1.002 1.002 0 000 1.437 1.047 1.047 0 001.463 0l.984-.966c.967-.95 2.542-1.135 3.602-.288a2.54 2.54 0 01.203 3.81l-2.903 2.852a2.646 2.646 0 01-3.696 0l-1.11-1.09L9 13.57l1.108 1.089c1.822 1.788 4.802 1.788 6.622 0l2.905-2.852a4.558 4.558 0 00-.357-6.82c-1.893-1.517-4.695-1.226-6.422.47"></path><path d="M11.144 19.543l.937-.92a1.002 1.002 0 000-1.437 1.047 1.047 0 00-1.462 0l-.985.966c-.967.95-2.542 1.135-3.602.288a2.54 2.54 0 01-.203-3.81l2.903-2.852a2.646 2.646 0 013.696 0l1.11 1.09L15 11.43l-1.108-1.089c-1.822-1.788-4.802-1.788-6.622 0l-2.905 2.852a4.558 4.558 0 00.357 6.82c1.893 1.517 4.695 1.226 6.422-.47"></path></g></svg></span></span></span><span class="smart-link-title-wrapper css-0">Discord - A New Way to Chat with Friends &amp; Communities</span></span></a></span></div></a> (Patron Discord)</p>
</div>

From Trello: https://trello.com/c/wayqgqSe

**Comment by Spidy123222** [comment]
> Is this feature still possible? If so, is it still planned?

Yes it's still possible and is in plans in some form. It's just not high in priority at the moment as other issues has taken priority.

**Comment by Awii2** [comment]
Been almost a year, I assume this is still not in priority? 

Just wondering if it’s still planned at all.

**Comment by CelloSerenity** [comment]
I believe closed by #1012.
